docs/prd/eks-kubeconfig.md

```diff
@@ -0,0 +1,619 @@
+    }
+
+    client := eks.NewFromConfig(cfg)
+
+    output, err := client.DescribeCluster(ctx, &eks.DescribeClusterInput{
+        Name: aws.String(clusterName),
+    })
+    if err != nil {
+        return nil, fmt.Errorf("failed to describe cluster: %w", err)
+    }
+
+    return &EKSClusterInfo{
+        Name:                     clusterName,
+        Endpoint:                 aws.ToString(output.Cluster.Endpoint),
+        CertificateAuthorityData: aws.ToString(output.Cluster.CertificateAuthority.Data),
+        ARN:                      aws.ToString(output.Cluster.Arn),
+        Region:                   region,
+    }, nil
+}
+```
+
+### Kubeconfig Generation
+
+#### Output Format
+
+The generated kubeconfig follows the standard format produced by `aws eks update-kubeconfig`:
+
+```yaml
+apiVersion: v1
+kind: Config
+current-context: dev-eks
+clusters:
+- name: arn:aws:eks:us-east-2:123456789012:cluster/dev-cluster
+  cluster:
+    server: https://XXXXXXXXXXXX.gr7.us-east-2.eks.amazonaws.com
+    certificate-authority-data: LS0tLS1CRUdJTi...
+contexts:
+- name: dev-eks
+  context:
+    cluster: arn:aws:eks:us-east-2:123456789012:cluster/dev-cluster
+    user: user-dev-cluster
+users:
+- name: user-dev-cluster
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1beta1
+      command: aws
+      args:
+      - eks
+      - get-token
+      - --cluster-name
+      - dev-cluster
+      - --region
+      - us-east-2
+```
+
+#### Exec Credential Plugin
+
+The kubeconfig uses the `aws eks get-token` exec plugin for authentication:
+
+- **Command**: `aws eks get-token --cluster-name <name> --region <region>`
+- **API Version**: `client.authentication.k8s.io/v1beta1`
+- **Token Lifetime**: ~15 minutes (refreshed automatically by kubectl on expiration)
+- **Credential Source**: Uses ambient AWS credentials (from Atmos-managed files)
+
+**Note**: We use the AWS CLI exec plugin rather than embedding Atmos as the exec plugin because:
+1. It follows established patterns users expect
+2. AWS CLI handles token caching and refresh
+3. No dependency on Atmos binary being in PATH for kubectl
+
+### File Path Management
+
+#### XDG-Compliant Default Path
+
+```go
+// Default: ~/.config/atmos/kube/config
+func GetDefaultKubeconfigPath() (string, error) {
+    kubeDir, err := xdg.GetXDGConfigDir("kube", 0700)
+    if err != nil {
+        return "", err
+    }
+    return filepath.Join(kubeDir, "config"), nil
+}
+```
+
+#### Environment Variable Integration
+
+After writing kubeconfig, users can set `KUBECONFIG` to use it:
+
+```bash
+export KUBECONFIG=$(atmos auth env --format=value KUBECONFIG)
+kubectl get pods
+```
+
+Or Atmos can set it automatically during `atmos auth shell`:
+
+```bash
+atmos auth shell
+# KUBECONFIG is set to Atmos-managed path
+kubectl get pods
+```
+
+### CLI Command
+
+#### Command: `atmos auth eks-kubeconfig`
+
+```
+atmos auth eks-kubeconfig [integration] [flags]
```

Comment:
I would rather not leak AWS things into the `atmos auth` command namespace. 
We have the `atmos aws` command namespace we can use instead. In fact, it already has a command for this we should/would update.